release-information/release-notes/6_0_0/6_0_0-release-notes-core-changes.md

```diff
@@ -16,6 +16,20 @@ The following new features were added for the {{site.productname}} 6.0 release.
 
 ### Feature name
 
+### Print plugin moved to TinyMCE Core
+
+The Print plugin has been moved to Core in {{site.productname}} 6.0.
+
+The print toolbar button, menu item, shortcut, and command will now work by default, without needing to specify `print` as a plugin.
```

Comment:
```suggestion
The `print` toolbar button, menu item, shortcut, and command will now work by default, without needing to specify `print` as a plugin.
```
See previous comment about this?

Reply:
Changed this to the same as the previous comment, and discussed with Tyler.